src/leader_scheduler.rs

```diff
@@ -19,10 +19,11 @@ use solana_vote_signer::rpc::LocalVoteSigner;
 use std::io::Cursor;
 use std::sync::Arc;
 
-pub const DEFAULT_BOOTSTRAP_HEIGHT: u64 = 1000;
-pub const DEFAULT_LEADER_ROTATION_INTERVAL: u64 = 100;
-pub const DEFAULT_SEED_ROTATION_INTERVAL: u64 = 1000;
-pub const DEFAULT_ACTIVE_WINDOW_LENGTH: u64 = 1000;
+pub const BLOCK_TICK_COUNT: u64 = 4;
```

Comment:
These names still don't make much sense to me. BLOCK_TICK_COUNT could be TICKS_PER_BLOCK. 
 DEFAULT_BOOTSTRAP_HEIGHT could be BOOTSTRAP_TICK_HEIGHT.  DEFAULT_LEADER_ROTATION_INTERVAL could be TICKS_PER_SLOT. DEFAULT_SEED_ROTATION_INTERVAL could be BLOCKS_PER_EPOCH, and is that the same thing as DEFAULT_ACTIVE_WINDOW_LENGTH?

Reply:
I can change the BLOCK_TICK_COUNT to TICKS_PER_BLOCK.
For others, I think we should wait for input from @carllin